src/Compilers/Core/Portable/SourceGeneration/GeneratorDriver.cs

```diff
@@ -41,13 +41,13 @@ internal GeneratorDriver(GeneratorDriverState state)
 
         internal GeneratorDriver(Compilation compilation, ParseOptions parseOptions)
```

Comment:
I'd expect to have a constructor that can a list of generators so we don't have to immediately `With` it in.

Reply:
Added.

---
In reply to: [387332242](https://github.com/dotnet/roslyn/pull/42030#discussion_r387332242) [](ancestors = 387332242)